build/build-bundle.js

```diff
@@ -1,7 +1,7 @@
 /**
- * @license Copyright 2018 The Lighthouse Authors. All Rights Reserved.
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ * @license
+ * Copyright 2018 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Changing the source file comments is fine, but the bundle is a form of distribution so it should retain the full reference to the license IMO.

Reply:
based on my read of the above docs, the minimal header would be fine for a bundle.  but... i don't care too much. i'm happy to keep the full text version for this case. :)

Reply:
I suppose the "indentifer" thing there is pretty cut and dry in terms of declaring the license that applies. Still, I think it's nice to give the whole shebang for the bundle

Reply:
lol we both thought this was the bundle comment header, but that's something else and less verbose. nvm